### ISSUE TITLE
What is the purpose of custom BatchNormalization in common.py?

### ISSUE THREAD

**Issue by lostmsu** [hunglc007/tensorflow-yolov4-tflite#265]
I am trying to figure out if it makes any sense for the training loop.

As far as I can see, `freeze_all`, that sets the `trainable` property, is only called on convolutional layers, so at least during training the new `BatchNormalization` has the same behavior as the original `BatchNormalization` from Keras, because its `trainable` is actually always `True`.

Can somebody clear my confusion here?
